apple/internal/templates/ios_sim.template.py

```diff
@@ -417,7 +418,14 @@ def extracted_app(ios_application_output_path, app_name):
   """
   if os.path.isdir(ios_application_output_path):
     logger.debug("Found app directory: %s", ios_application_output_path)
-    yield os.path.realpath(ios_application_output_path)
+    with tempfile.TemporaryDirectory(prefix="bazel_temp") as temp_dir:
+      temp_app_path = os.path.join(temp_dir, app_name + ".app")
+      shutil.copytree(ios_application_output_path, temp_app_path)
+      for root, dirs, _ in os.walk(temp_app_path):
+        for directory in dirs:
+          os.chmod(os.path.join(root, directory), 0o777)
```

Comment:
Is it faster to just blanket apply 0x777 to all entities recursively? Does it break anything?

Reply:
I assume there will be more files than directories so that would end up doing a lot more `chmod` calls. I imagine it _could_ break something but I'm not sure how risky it actually would be.

Reply:
Ahh, there isn't a `chmod -R` version, never mind.